modules/installation-osp-local-disk-deployment.adoc

```diff
@@ -154,13 +154,12 @@ done
 <3> Outputs the name of every control plane machine which has an `additionalBlockDevice` named `etcd`.
 
 . Create a file named `98-var-lib-etcd.yaml` by using the following YAML file:
-
++
```

Comment:
**issue:** The Procedure heading get mixed with warning. Screencap from the preview I see.
![image](https://github.com/openshift/openshift-docs/assets/23069445/4b576a41-8f03-427e-aa80-3dd215f093aa)

**suggestion:** I would probably rethink this and make following changes to fix this issue:
1. Add an introduction to the module https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs
2. Add the admonition after the introduction. `This procedure is for testing etcd on a local disk and should not be used on a production cluster. In certain cases, complete loss of the control plane can occur.`
3. Add the `see "Overview of backup and restore operation" under "Backup and restore.` part under the additional resources section on the assembly with proper links.

Reply:
That fix is outside of the scope of what this PR is intended to do, but I'll roll it in, sure.